src/components/Alert.ts

```diff
@@ -1,8 +1,8 @@
-import { DOM, SFC } from "react";
+import { SFC, createElement } from "react";
 
 const Alert: SFC<{ message?: string }> = ({ message }) =>
```

Comment:
Make alert generic so that it can be used for different kind of alert like success, info, etc.